run-r-script/test.cwl

```diff
@@ -9,9 +9,14 @@ hints:
     #     $include: ./Dockerfile
     dockerPull: rocker/tidyverse:4.1
 requirements:
-  EnvVarRequirement:
-    envDef:
-      ROOT: "true"
+  - class: NetworkAccess
+    networkAccess: true  
+  - class: InlineJavascriptRequirement
+  - class: InitialWorkDirRequirement
+    listing:
+      - entry: "$({class: 'Directory', listing: []})"
+        entryname: "./lib"
+        writable: true
```

Comment:
This might be replaceable by:
``` CWL
    InitialWorkDirRequirement:
        listing:
            - class: Directory
              basename: lib
              listing: []
```
Which would probably also result in
```CWL
  - class: InlineJavascriptRequirement
```
not being needed anymore.

Reply:
I tried that as well, but as far as I can see [Directory](https://www.commonwl.org/v1.0/CommandLineTool.html#Directory) has no `writable` field, whereas there is one for [Dirent](https://www.commonwl.org/v1.0/CommandLineTool.html#Dirent). If there is a possibility to make it writable it would definitely be cleaner.

Reply:
It should be writable because the directory gets created in $runtime.outdir and from my tests in https://github.com/nfdi4plants/cwl-examples/pull/4 it worked. Maybe something is different there?

Reply:
I think cwl handles written files differently, depending on where the file is written. If it is an output for a workflow it usually works, but as soon as it happens during a workflow and is not the output, different permissions are required. When I try this:
```
requirements:
  NetworkAccess
    networkAccess: true  
  InitialWorkDirRequirement:
    listing:
      - class: Directory
        basename: lib
        listing: []
```
I get the following error:
![image](https://user-images.githubusercontent.com/39764934/133431009-75aa5cbf-f568-4ad2-990e-81dc3c1fee81.png)

Reply:
Good catch! I will have to rework https://github.com/nfdi4plants/cwl-examples/pull/4 then!